spec/close-tag-spec.coffee

```diff
@@ -26,11 +26,22 @@ describe 'closeTag', ->
       tags = tagFinder.tagsNotClosedInFragment(fragment)
       expect(tags).toEqual(['html', 'body', 'h1'])
 
+    # Is this desirable? It results in malformed XML for closing tag insertions.
```

Comment:
The reason for the behavior asserted in the following spec was not obvious to me, so I added a comment that I thought would be useful for future coders, or would prompt a knowledgeable dev to consider it (and either clarify the requirement or change the behavior).